src/main/java/org/kpmp/repositoryDataset/RepositoryFileDataset.java

```diff
@@ -229,12 +231,42 @@ public void setTissueSource(String tissueSource) {
 
     @Nullable
     @JsonProperty("release_version")
-    public Integer getReleaseVersion(){
+    public String getReleaseVersion(){
         return releaseVersion;
     }
 
-    public void setReleaseVersion(Integer releaseVersion){
+    public void setReleaseVersion(String releaseVersion){
         this.releaseVersion = releaseVersion;
     }
 
-}
\ No newline at end of file
+    @Transient
+    private String participantIdSort;
+    @Transient
+    private String fileNameSort;
+    @Transient
+    private String platformSort;
+
+    @JsonProperty("participant_id_sort")
+    public String getParticipantIdSort(){
+        
```

Comment:
Do we need to worry about a file with 0 participant ids?
Also, upon looking at the view this is working with, I don't think this is going to work the way you want. When I search for the clinical data file, I don't see comma separated participant ids, I see lots of entries for the same file. So, with the way this is written, we would have the clinical data file listed a million times.